### PR TITLE
Added dataloader for Panoptic Mask R-CNN (#10112)

### PR DESCRIPTION
**Pull request by jaeyounkim**
# Description

Contribution from @srihari-humbarwadi 

https://github.com/tensorflow/models/pull/10112

* Added `panoptic_maskrcnn_input.Parser`
* Added tests for `panoptic_maskrcnn_input.Parser`
* added `DataConfig`
* fixed doc-string
* Added type annotations

### CODE REVIEW

**Review by arashwan — APPROVED** [review]

**Comment by srihari-humbarwadi** [comment]
@googlebot I consent
